### PR TITLE
Card

### PR DESCRIPTION
**Pull request by surbhi84**

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -20,19 +20,19 @@ body {
...
 }
 
-.clr-yp {
+.color-yp {
   background-color: var(--yellow-pastel);
 }
 
-.clr-bp {
+.color-bp {
   background-color: var(--blue-pastel);
```
Comment (anupkgurung) [inline components/alert/alert.css:36]: what is

Comment (anupkgurung) [inline components/alert/alert.css:36]: what is bp, yp ,rp ?

**Review by anupkgurung — APPROVED** [review]
Looks good. If rp,bp are from some other widely used class names than it great else try to name a bit more specific.

File: components/cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  background-color: rgba(255, 255, 255, 0.151);
+}
+
+.text-only {
+  height: 12rem;
+}
+
+.hrz-card {
+  display: flex;
+  align-items: center;
```
Comment (aniketsingh13) [inline components/cards/cards.css:74]: if flex uses more in the code you can make utility class for it

**Review by aniketsingh13 — APPROVED** [review]
consider the changes otherwise good to go

**Review by maitrakhatri — APPROVED** [review]
looks good to me

File: components/alert/alert.css (hunk 1 above)
Comment (surbhi84) [reply components/alert/alert.css:36]: rp as in red pastel ,yellow pastel and so on i'hv added color to the class name that signifies it's a color, these colors are also defined in my variables so wherever they'll be used i will use the same naming convention.

File: components/cards/cards.css (hunk 2 above)
Comment (surbhi84) [reply components/cards/cards.css:74]: Thanks i'll consider it.

**Comment by surbhi84** [comment]
Thankyou everyone for approving.

**Pull request merged**